### ISSUE TITLE
Can play hls video in Browser Android emulate mode?

### ISSUE THREAD

**Issue by syagawa** [videojs/videojs-contrib-hls#1305]
## Description
Hls video can not be played in browser's Android emulated mode.
It is not a big problem because it can be played on an actual Android device.
But, I want to know if the fact that videojs-contrib-hls does not work in Android emulated mode on browser is a correct behavior or a bug.

## Steps to reproduce
1. Open the Android emulator mode with Windows Chrome's developer tool.
2. Go to "https://videojs.github.io/videojs-contrib-hls/".
3. And if you try to play video, you get an error.

### Expected
It plays without a problem

### Error output
```
VIDEOJS: ERROR: (CODE: 4 MEDIA_ERR_SRC_NOT_SUPPORTED)" The media could not be loaded, either because the server or network failed or because the format is not supported.
```

## Additional Information
It can not play even in FireFox's Android emulated mode.

It can play in Chrome's iOS emulated mode.
It can play in Chrome on my Android 6.0 device.

### videojs version
video.js 6.4.0

### Browsers
Chrome 63.0, Firefox 57.0.2

### Platforms
Windows 7 pro 64bit

**Comment by forbesjo** [comment]
I do not see any problems with using an emulated Pixel 2 or Galaxy S5. If this is still a problem please let us know

**Comment by syagawa** [comment]
Thank you for your reply.

I looked at this page (https://videojs.github.io/videojs-contrib-hls/) again.
Of course it's in Android emulated mode on Chrome(Windows 7).

Then, no error appears.
Chrome was able to play video.

After checking the version of videojs, it was displayed as 6.7.3.
The videojs version when I posted this issue was 6.4.0.

Checking the changelog of video.js, we found a commit https://github.com/videojs/video.js/commit/f03ac5e .

I have not confirmed the details, but probably the issue was fixed with this commit.

Thank you.
